gulpfile.js

```diff
@@ -705,33 +705,39 @@ gulp.task('testLib_NodeModules', gulp.series('testLib', function () {
 gulp.task('testResources', gulp.parallel('testLib_NodeModules', 'ps1tests', 'tstests', 'copyTestData'));
 
 gulp.task("test", gulp.series("testResources", function(){
-    process.env['TASK_TEST_TEMP'] =path.join(__dirname, _testTemp);
+    process.env['TASK_TEST_TEMP'] = path.join(__dirname, _testTemp);
+
     shell.rm('-rf', _testTemp);
     shell.mkdir('-p', _testTemp);
 
-    if (options.suite.indexOf("ArtifactEngine") >= 0  && options.e2e) {
-        var suitePath = path.join(_testRoot, "Extensions/" + options.suite + "/**/*e2e.js");
+    if (options.suite.indexOf("ArtifactEngine") >= 0 && options.e2e) {
+        const suitePath = path.join(_testRoot, "Extensions/" + options.suite + "/**/*e2e.js");
         console.log(suitePath);
-        var tfBuild = ('' + process.env['TF_BUILD']).toLowerCase() == 'true'
-        return gulp.src([suitePath])
+        const tfBuild = process.env['TF_BUILD'].toLowerCase() == 'true';
+        return gulp
+            .src([suitePath])
             .pipe(mocha({ reporter: 'spec', ui: 'bdd', useColors: !tfBuild }));
     }
 
     if (options.suite.indexOf("ArtifactEngine") >= 0  && options.perf) {
-        var suitePath = path.join(_testRoot, "Extensions/" + options.suite + "/**/*perf.js");
+        const suitePath = path.join(_testRoot, "Extensions/" + options.suite + "/**/*perf.js");
         console.log(suitePath);
-        var tfBuild = ('' + process.env['TF_BUILD']).toLowerCase() == 'true'
-        return gulp.src([suitePath])
+        const tfBuild = process.env['TF_BUILD'].toLowerCase() == 'true';
+        return gulp
+            .src([suitePath])
             .pipe(mocha({ reporter: 'spec', ui: 'bdd', useColors: !tfBuild }));
     }
 
-    var suitePath = path.join(_testRoot,"Extensions/" + options.suite + "/Tests/Tasks", options.suite + '/_suite.js');
+    const suitePath = path.join(_testRoot,"Extensions/" + options.suite + "/Tests/Tasks", options.suite + '/_suite.js');
     console.log(suitePath);
-    var suitePath2 = path.join(_testRoot, "Extensions/" + options.suite + "/**/*Tests.js");
+    const suitePath2 = path.join(_testRoot, "Extensions/" + options.suite + "/**/*Tests.js");
     console.log(suitePath2);
-    var tfBuild = ('' + process.env['TF_BUILD']).toLowerCase() == 'true'
-    var ignorePath = "!" + path.join(_testRoot, "Extensions",  "/**/UIContribution{,/**}");
-    return gulp.src([ suitePath, suitePath2, ignorePath ], { allowEmpty: true })
+
+    const tfBuild = ('' + process.env['TF_BUILD']).toLowerCase() == 'true'
```

Comment:
Why do we still use ('' + ..) here but above such cases were removed?

Reply:
On line 716 now it's:
tfBuild = process.env['TF_BUILD'].toLowerCase() == 'true';
previously it was:
tfBuild = ('' + process.env['TF_BUILD']).toLowerCase() == 'true'

Another case - line 725 - now it's:
tfBuild = process.env['TF_BUILD'].toLowerCase() == 'true';
previously it was:
tfBuild = ('' + process.env['TF_BUILD']).toLowerCase() == 'true'

So I prefer to have consistent changes - we either preserve  ('' + ..) everywhere, or remove it in all places.